### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by tyson-dev** [blizzhackers/documentation#16] [General]
https://github.com/blizzhackers/documentation/blob/master/kolbot/MiscellaneousOptions.md#lifechicken-restart-profile

Restart after life chicken.

Can this please be updated as there is no Line 564 in the linked toolsthread.js

Maybe add the Line that has to be replaced so it can be found in future by hand.

TY

**Comment by theBGuy** [comment]
There is and has been quite a bit of changes going on, when everything is more concrete I'll come back through and update everything here.
